src/theme/ThemeContext.js

```diff
@@ -1,21 +1,19 @@
 // @flow
 
-import { cacheStyles as cacheStylesInner, makeThemeContext } from 'react-native-patina'
+import { makeThemeContext } from 'react-native-patina'
 
 import { type Theme } from '../types/Theme.js'
 import { edgeDark } from './variables/edgeDark.js'
 
-// Provide the theme context methods:
-const themeContext = makeThemeContext(edgeDark)
-export const { ThemeProvider, useTheme, withTheme, changeTheme, getTheme, watchTheme } = themeContext
+export type { Theme }
 
+/**
+ * Utility type for declaring `withTheme` components.
+ */
 export type ThemeProps = {
   theme: Theme
 }
 
-/**
- * Wrap cacheStyles with our specific Theme type.
- */
-export function cacheStyles<Styles: Object>(callback: (theme: Theme) => Styles): (theme: Theme) => Styles {
-  return cacheStylesInner(callback)
-}
+// Provide the theme context methods:
+const themeContext = makeThemeContext(edgeDark)
+export const { ThemeProvider, useTheme, withTheme, changeTheme, getTheme, watchTheme } = themeContext
```

Comment:
Suggested to re-add `cacheStyles` to be re-added to the exports so all Theme related is can be imported all using this file

Reply:
Yeah, that makes sense. Done.